pareto/operational_water_management/operational_produced_water_optimization_model.py

```diff
@@ -884,152 +1194,318 @@ def create_model(df_sets, df_parameters, default={}):
     # COMMENT: Remove pipeline/disposal/storage capacity expansion increment parameters
     model.p_delta_Pipeline = Param(
         model.s_D,
-        default=10,
+        default=pyunits.convert_value(
+            10000,
+            from_units=pyunits.oil_bbl,
+            to_units=model.model_units["volume"],
+        ),
         initialize=PipelineCapacityIncrementsTable,
-        doc="Pipeline capacity installation/expansion increments [bbl/day]",
+        units=model.model_units["volume_time"],
+        doc="Pipeline capacity installation/expansion increments [volume/time]",
     )
     model.p_delta_Disposal = Param(
         model.s_I,
-        default=10,
+        default=pyunits.convert_value(
```

Comment:
this is an inconsistency, you are correct. It wasn't caught in our tests because the operational model actually never uses this variable as it's related to disposal capex, something we don't consider in operational model. I've updated for consistency purposes, but this variable likely needs to be removed eventually.